_config.yml

```
@@ -959,3 +959,10 @@ podcast:
           url: "https://artsy.github.io/blog/2016/01/30/iOS-Junior-Interviews/"
         - title: "Artsy jobs"
           url: "https://www.artsy.net/jobs"
+    - title: "3: WAYAHDYGH: Chris"
+      date: "2021-01-19 16:31:14 -0500"
```

```suggestion
      date: "2021-01-20 16:31:14 -0500"
```

This date ends up showing as the publish date within podcast players. Should we try to make these always have a Wednesday date so it looks like we always publish on Wednesdays? (If so I can add something in a followup that pre-populates this with the next Wednesday instead of today)

---

oh, woops! Now I see your comment about Thursday - I guess I forgot our cadence 🤪 

So same comment, but for Thursdays: 

```suggestion
      date: "2021-01-21 16:31:14 -0500"
```

---

I really don't want to have this go out on inauguration day -- ideally it could go out next week

---

done!